azerothcore/azerothcore-wotlk
# fix(DB/creature_loot_template): Adjust drop chance for white items in starter zones

Ported from https://github.com/TrinityCore/TrinityCore/issues/17578

## Changes Proposed:
-  reduce the drop chances for several white items  

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4040
 
## Tests Performed:
- I didn't test


## How to Test the Changes:
I don't think it is really needed to test.
But to test it you just need to check which NPCs drop one of the following items and see if you can drop easily it (you shouldn't).
```
1438, 2959, 2960, 2963, 2964, 2967, 2972, 2975, 3200, 3207, 3213, 3214, 3323, 3643, 4560, 4658, 4662, 4684, 4690, 6506, 6508, 6514, 6515, 6517, 6549, 6566, 6579, 7350, 7351, 9744, 9745, 9750, 9752, 9754, 9759, 9760
```

## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
